ui/prometheus-plugin/src/plugins/prometheus-time-series-query/get-time-series-data.ts

```diff
@@ -105,6 +105,7 @@ export const getTimeSeriesData: TimeSeriesQueryPlugin<PrometheusTimeSeriesQueryS
     }),
     metadata: {
       notices,
+      resolvedQuery: query,
```

Comment:
Some extra metadata that plugins can set that has the resolvedQuery string with interpolated variables. Similar to Grafana's `exectedQueryString`
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/data.ts#L85
Probably should rename to that?

Reply:
No strong feelings about the name, but this is another one that could probably benefit from a little jsdoc above the type def to provide a little more context for consumers about what it contains and why they might want to use it.

Reply:
Decided to use the same thing as grafana `executedQueryString` (no good reason to be different IMO). I also added a jsdoc comment